curriculum/challenges/english/25-front-end-development/lecture-introduction-to-javascript-libraries-and-frameworks/6734e3a9cc78faaf4248d335.md

```diff
@@ -92,7 +87,7 @@ COBOL
 
 ### --feedback--
 
-Review the beginning of the video where React libraries were first discussed.
+Review the section where React libraries were first discussed.
```

Comment:
Let's make this more specific. like "reviewing the beginning section" or something like that. Apply to all feedbacks